java/code/src/com/suse/manager/webui/controllers/VirtualHostManagerController.java

```diff
@@ -616,8 +616,7 @@ public static Object refresh(Request request, Response response, User user) {
             Map<String, String> params = new HashMap<>();
             params.put(GathererJob.VHM_LABEL, label);
             try {
-                new TaskomaticApi()
-                        .scheduleSingleSatBunch(user, "gatherer-matcher-bunch", params);
+                new TaskomaticApi().scheduleGathererRefresh(user, params);
```

Comment:
If the label is the only possible value which can be set I would also go with @hustodemon suggestion to replace "params" with "label" and create the params map inside of the method.

But for me this is optional. Not sure what franky think :-)

Reply:
Yes, my comments weren't addressed yet. @mseidl let me know if there is anything unclear about them.